liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/BasicSupport.java

```diff
@@ -312,7 +314,7 @@ protected void init() throws MojoExecutionException, MojoFailureException {
             // Set server directory
             serverDirectory = new File(serversDirectory, serverName);
             
-            log.info(MessageFormat.format(messages.getString("info.variable.set"), "serverDirectory", serverDirectory));
+            initLog.info(MessageFormat.format(messages.getString("info.variable.set"), "serverDirectory", serverDirectory));
```

Comment:
Yeah.  I pushed another commit which addresses this.   It has a bit of a downside in that it reintroduces the original problem of logging stuff that's not necessarily relevant, e.g. we might happen to hit a failure stepping through the logic for an upstream module we're not really going to install based on.   But we'd have to do a bigger refactor to address that.

Reply:
I guess another thought is:  we could imagine changing all the log.info(s) to log.debug(s) within init(), and then logging only some summary messages in installServerAssembly().

That would require more thought to confirm we were both:
*  adding all relevant info on every path
* AND NOT taking away something useful though by having the debug/info messages interleaved exactly as they are during the current execution flow.

So that's why I went this route.. but if it seems too unwieldy and the other approach better that could work too.